kolibri/plugins/learn/assets/src/views/LearnAppBarPage.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+
+  <AppBarCorePage :title="appBarTitle" :applyStandardLayout="applyStandardLayout">
+
+    <template #subNav>
+      <LearnTopNav ref="topNav" />
+    </template>
+
+    <div v-if="!loading" :style="styles">
+      <slot></slot>
+    </div>
+    <KLinearLoader
+      v-if="loading"
+      class="loader"
+      type="indeterminate"
+      :delay="false"
+    />
+
+  </AppBarCorePage>
+
+</template>
+
+
+<script>
+
+  import { mapState } from 'vuex';
+  import AppBarCorePage from 'kolibri.coreVue.components.AppBarCorePage';
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import LearnTopNav from './LearnTopNav';
+
+  export default {
+    name: 'LearnAppBarPage',
```

Comment:
I'd suggest not using `_Page` in layout components names to make their purpose clear and to allow ourselves easily differentiate between layout components and real pages connected to routes. Maybe `LearnAppBarLayout`?